src/Module.Client/GUI/CrpgMissionMarkerVM.cs

```diff
@@ -63,6 +67,11 @@ public CrpgMissionMarkerVm(Camera missionCamera, MissionMultiplayerGameModeBaseC
             }
         }
 
+        if (_commanderBehaviorClient != null)
+        {
+            _commanderBehaviorClient.OnCommanderUpdated += OnCommanderUpdated;
+        }
+
```

Comment:
_commanderBehaviorClient?.OnCommanderUpdated += OnCommanderUpdated;

Reply:
Event subscriptions don't allow for null-conditional operators to be used for some reason. You receive the following error: `CS0079 the event ' _commanderBehaviorClient.OnCommanderUpdated' can only appear on the left hand side of += or -=`.

Our existing implementations for event subscriptions are using this technique for null-checking.